change/@fluentui-react-badge-3bd9df4e-4e19-4ff6-b2bf-accfb6602308.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "fix: Match OOF icon mapping to v8.",
```

```suggestion
  "comment": "fix: Presence badge should show Out of Office icon for away+OOF and Offline+OOF statuses.",
```
We should be more descriptive in the change